generator/mavparse.py

```diff
@@ -456,7 +456,7 @@ def __str__(self):
 def message_checksum(msg):
     '''calculate CRC-16/MCRF4XX checksum of the key fields of a message, so we
        can detect incompatible XML changes'''
-    from .mavcrc import x25crc
+    from pymavlink.generator.mavcrc import x25crc
```

Comment:
how is this circular? mavcrc.py doesn't include this module

Reply:
I believe the cause of the issue is I had a partially initialized module due to another python module of the same name.

This fixes it though and is more robust anyways in my opinion 

`AttributeError: partially initialized module 'mavcrc' has no attribute 'x25crc' (most likely due to a circular import)`